kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -281,6 +281,42 @@ class PrometheusJMXExporterConfig(CamelCaseConfigModel, DescConfigModel):
     )
 
 
+class JMXRule(SerializeAsOptionalModel, CamelCaseConfigModel, DescConfigModel):
+    """JMX rule.
+
+    :param pattern: The rule pattern.
+    :param type: Type of the rule.
+    :param name: Name of the rule.
+    :param help: Help for the rule.
+    :param labels: Labels for the rule.
+    """
+
+    pattern: str | None = Field(
+        default=None,
+        description=describe_attr("pattern", __doc__),
+    )
+
+    type: str | None = Field(
```

Comment:
Seems to be an enum https://github.com/prometheus/jmx_exporter/blob/main/integration_test_suite/integration_tests/src/main/java/io/prometheus/jmx/test/support/metrics/Metric.java#L33